src/components/about.js

```diff
@@ -0,0 +1,126 @@
+import second from './about/img/second.svg';
+import third from './about/img/third.svg';
+import fourth from './about/img/fourth.svg';
+import './about/css/about.css';
+
+const aboutusinfo1 ="National Institute of Technology Patna is the 18th National Institute of Technology created by the Ministry of H.R.D. Government of India after rechristening the erstwhile Bihar College of Engineering Patna on 28. 01. 2004. NIT Patna marked its humble beginning in 1886 with the establishment of pleaders survey training school which was subsequently promoted of Bihar College of Engineering Patna in 1924. This made this institute the 6th Oldest Engineering Institute of India. The graduate level curriculum was later elevated to the post graduate level in 1978. The institute is situated on the south bank of holy river Ganges behind Gandhi Ghat, one of the most important and reverential place of Patna. The Gandhi Ghat is associated with the immersion of ashes of father of the Nation Mahatma Gandhi in the river Ganges. The campus has a picturesque river view with historic building presenting a spectacle of architectural delight and natural beauty.";
+const aboutusinfo2 ="National Institute of Technology Patna has been declared as an Institute of National Importance and has been granted a fully Autonomous Status by MHRD, Government of India. The Institute has also been declared as a Centre of Excellence of impart high level education training , research and development in science, engineering technology and humanities. It is imparting high quality education & values at UG (B.Tech), PG (M.Tech) & Ph .D programmes through its experienced faculty well versed in their respective field of engineering an technology with well equipped laboratories . At present the Institute has seven disciplines viz. Architecture, Civil Engineering, Compute Science & Engg., Electrical Engg., Electronics & Communication Engg., Information Technology and Mechanical Engg., and well established departments of physics, Mathematics and Humanities and Social Science.";
+const aboutusinfo3 ="National Institute of Technology Patna aims at setting out very high education standards and holds long record of academic excellence. The pedagogical aspects have been formulated to suit not only the needs of the contemporary industrial requirements but also to develop human potential to its fullest extent in a range of professions. Extra curricular activities are planed through games and sports, cultural programmes and NSS activities. Cultural activities provide a platform to know about the culture of various states and regions of the country and opportunity for national integration.";
+const aboutusinfo4 ="Ever since its rechristening, NIT Patna has been on the fast track of development and has undergone numerous facelifts because of which placement records have witnessed unprecedented growth and is touching new heights as the graph of placement is increasing remarkably.";
```

Comment:
Can we group all about us info in a separate file/component ?
Same with mission infos, campus infos and values info

Reply:
Yes sir

Reply:
@supu2701 To make things even cleaner, what you could try putting all of the data inside these variables into one single array, and then use the array's map function to return a component for each element of the array. I'll explain how you can do that.

Suppose you have six constants with data as: 
```
const aboutusinfo1 = "data 1"
const aboutusinfo2 = "data 2"
const aboutusinfo3 = "data 3"
const aboutusinfo4 = "data 4"
const aboutusinfo5 = "data 5"
const aboutusinfo6 = "data 6"
```

What you could try here is to use one single array constant in this form:
```
const aboutusinfo = [
    "data 1",
    "data 2",
    "data 3",
    "data 4",
    "data 5",
    "data 6"
]
```

As already visible, six constants have been reduced to one, improving readability.

Now have a look at the other side, where you use these constants:

Currently, these have been used as:
```
<div className="row">
    <h1>About Us</h1>
</div>
<div className="row">
    <h3>{aboutusinfo1}</h3>
</div>
<div className="row">
    <h3>{aboutusinfo2}</h3>
</div>
<div className="row">
    <h3>{aboutusinfo3}</h3>
</div>
<div className="row">
    <h3>{aboutusinfo4}</h3>
</div>
```

With an array variable, you could simply write:
```
<div className="row">
    <h1>About Us</h1>
</div>
{
    aboutusinfo.map((elem) => {
        return <div className="row">
                <h3>{elem}</h3>
              </div>
    })
}
```

With this design, maintaining the data becomes easier, as you do not need to add a `<div><h3></h3></div>` every time you add a new element in the data array.

This is not something that one should necessarily use, but it does make maintenance a lot easier. I'll suggest you consider using it if you understood it. If you have any doubts, feel free to ask me.